src/smart-components/role/RolesTable.tsx

```diff
@@ -118,6 +135,23 @@ const RolesTable: React.FunctionComponent<RolesTableProps> = ({ selectedRole = u
     <React.Fragment>
       <ContentHeader title="Roles" subtitle={''} />
       <PageSection isWidthLimited>
+        {isDeleteModalOpen && (
+          <WarningModal
+            ouiaId={`${ouiaId}-remove-role-modal`}
+            isOpen={isDeleteModalOpen}
```

Comment:
Based on the mocks the modal is missing checkbox
```suggestion
            isOpen={isDeleteModalOpen}
            withCheckbox
```